drivers/include/digit7seg.h

```diff
@@ -0,0 +1,146 @@
+/*
+ * Copyright (C) 2024 Orange
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     drivers_digit7seg
+ * @brief       Device driver for less than 5 digits of 7 segments without IC
+ * @file
+ * @author      Pierre Le Meur <pierre1.lemeur@orange.com>
+ * @{
+ *
+ */
+
+#ifndef DIGIT7SEG_H
+#define DIGIT7SEG_H
+
+#include <inttypes.h>
+
+#include "macros/units.h"
+#include "periph/gpio.h"
+#include "periph/timer.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#define DIGIT7SEG_MAX_DIGITS  (4)     /**< Max pin number, can be below or equal */
+#ifndef DIGIT7SEG_TIMER_HZ
+#   define DIGIT7SEG_TIMER_HZ    MHZ(1)  /**< default timer hz */
```

Comment:
```suggestion
#  define DIGIT7SEG_TIMER_HZ    MHZ(1)  /**< default timer hz */
```
as per the [Coding Convention](https://github.com/RIOT-OS/RIOT/blob/master/CODING_CONVENTIONS.md#indentation-of-preprocessor-directives). please also check other places